lib/processCss.js

```diff
@@ -77,30 +77,55 @@ var parserPlugin = postcss.plugin("css-loader-parser", function(options) {
 			exports[exportName] = replaceImportsInString(exports[exportName]);
 		});
 
+		function processNode(item, isValue) {
+			switch (item.type) {
+				case "value":
+					item.nodes.forEach(function(node){
+						processNode(node, true);
+					});
+					break;
+				case "nested-item":
+					item.nodes.forEach(function(node){
+						processNode(node);
+					});
+					break;
+				case "item":
+					var importIndex = imports["$" + item.name];
+					if (typeof importIndex === "number") {
+						item.name = "___CSS_LOADER_IMPORT___" + importIndex + "___";
+					}
+					break;
+				case "url":
+					if (!/^#/.test(item.url) && loaderUtils.isUrlRequest(item.url, options.root)) {
+						item.stringType = "";
+						delete item.innerSpacingBefore;
+						delete item.innerSpacingAfter;
+						var url;
+
+						if(isValue) {
```

Comment:
`isValue` is always `true` here, because a nested item has this representation:

``` js
{
	type: "nested-item",
	name: name,
	nodes: [
		{ type: "value", nodes: [...] }
	]
}
```

So I think you could remove this check here and always use the true case.